package.json

```
@@ -88,9 +86,10 @@
     "prettier": "^2.7.1",
     "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
+    "release-it": "^14.2.1",
+    "release-it-lerna-changelog": "^3.1.0",
```

```suggestion
    "@release-it-plugins/lerna-changelog": "^5.0.0",
```

The package was renamed, more info in the v5 release:
https://github.com/release-it-plugins/lerna-changelog/releases/tag/v5.0.0